app/components/app-monitoring/table/store-version/index.scss

```diff
@@ -1,6 +1,6 @@
 .production-version {
   letter-spacing: -0.0286em;
-  color: var(--appmonitoring-color-black);
+  color: var(--appmonitoring-color-ios);
```

Comment:
Version color should not depend on platform.. let it be `--appmonitoring-color-black` itself